testapp/src/main/res/layout/qa_settings_activity.xml

```diff
@@ -80,19 +53,41 @@
         <com.google.android.material.textfield.TextInputLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:padding="@dimen/small_8"
+            android:padding="@dimen/small_8">
 
-            android:visibility="visible">
-
-            <EditText
+            <androidx.appcompat.widget.AppCompatEditText
                 android:id="@+id/edtCustomErrorMessage"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/small_4"
                 android:hint="@string/hint_custom_error_message"
                 android:imeOptions="actionDone"
                 android:inputType="text" />
         </com.google.android.material.textfield.TextInputLayout>
 
+        <androidx.appcompat.widget.AppCompatTextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/bg_section"
+            android:paddingStart="@dimen/medium_16"
+            android:paddingTop="@dimen/small_4"
+            android:paddingEnd="@dimen/medium_16"
+            android:paddingBottom="@dimen/small_4"
+            android:text="@string/lb_unique_id" />
+
+        <com.google.android.material.textfield.TextInputLayout
```

Comment:
I think you should add a toggle switch for enabling the error, similar to how "Access Token" errors work.

Reply:
Yeah, I have added a toggle switch with this input view.